pkg/maps/policymap/policymap.go

```diff
@@ -471,6 +471,13 @@ func Create(path string) error {
 	return m.Create()
 }
 
+// Create unpinned creates a policy map without pinning it to the filesystem
+func CreateUnpinned(path string) (*PolicyMap, error) {
```

Comment:
Why would we want to do this? What does the path mean if not pinned?

Reply:
Yes. It's not a path at a first, but it will be once it is pinned.

Reply:
Unpinned maps still ned a path name for [the register](https://github.com/cilium/cilium/blob/main/pkg/bpf/map_register_linux.go#L20).